tests/unit/Service/SQS.service.test.js

```diff
@@ -33,31 +37,94 @@ const getService = ({ sendMessage = null, invoke = null } = {}, isOffline = fals
 
   jest.spyOn(service, 'sqs', 'get').mockReturnValue(sqs);
   jest.spyOn(service, 'lambda', 'get').mockReturnValue(lambda);
-  jest.spyOn(di, 'isOffline', 'get').mockReturnValue(isOffline);
 
   return service;
 };
 
 describe('Service/SQS', () => {
+  beforeAll(() => {
+    process.env.LAMBDA_WRAPPER_OFFLINE_SQS_MODE = undefined;
+  });
+
+  afterAll(() => {
+    delete process.env.LAMBDA_WRAPPER_OFFLINE_SQS_MODE;
+  });
```

Comment:
Should we use LAMBDA_WRAPPER_OFFLINE_SQS_MODE in other tests or in non-trivial ways, the current setup may break. When it comes to mock env variables, I usually store the current value before each test and restore it after each test:

```suggestion
  let envOfflineMode;
  
  beforeEach(() => {
    envOfflineMode = process.env.LAMBDA_WRAPPER_OFFLINE_SQS_MODE
  });

  afterEach(() => {
    process.env.LAMBDA_WRAPPER_OFFLINE_SQS_MODE = envOfflineMode;
  });
```

Reply:
Yeah, good suggestion. I assumed other tests that depend on the value would set it themselves. I had considered removing these hooks entirely but then forgot!

https://github.com/comicrelief/lambda-wrapper/pull/292/commits/f4a5ab0464524cd75cfc5bd817b133d02cfd4071